src/ConcreteOperations/intersection.jl

```diff
@@ -19,17 +17,10 @@ Return the intersection of a singleton with another set.
 If the sets intersect, the result is `S`.
 Otherwise, the result is the empty set.
 """
-function intersection(S::AbstractSingleton{N},
-                      X::LazySet{N}
-                     ) where {N<:Real}
-    return element(S) ∈ X ? S : EmptySet{N}(dim(S))
-end
+intersection(S::AbstractSingleton{N}, X::LazySet{N}) where {N<:Real}
```

Comment:
this can be probably further improved, such that Julia can actually document one of the methods (say the original method).

Reply:
Documentation seems to work in newer Julia versions, right?